jacktrip.pro

```diff
@@ -209,6 +209,8 @@ HEADERS += src/DataProtocol.h \
            src/CompressorPresets.h \
            src/Limiter.h \
            src/Regulator.h \
+           src/WaitFreeRingBuffer.h \
+           src/WaitFreeFrameBuffer.h \
```

Comment:
I think these need to get added to the meson.build file too?

Reply:
I wasn't sure if that was only for QOBJECT mocking or not, but I added them in just in case.